invenio/modules/workflows/upgrades/workflows_2015_06_05_resize_uuid_columns.py

```diff
@@ -0,0 +1,64 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of Invenio.
+# Copyright (C) 2015 CERN.
+#
+# Invenio is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License as
+# published by the Free Software Foundation; either version 2 of the
+# License, or (at your option) any later version.
+#
+# Invenio is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Invenio; if not, write to the Free Software Foundation, Inc.,
+# 59 Temple Place, Suite 330, Boston, MA 02111-1307, USA.
+
+"""Upgrade recipe."""
+
+from invenio.ext.sqlalchemy import db
+from invenio.modules.upgrader.api import op
+
+depends_on = [u'workflows_2014_08_12_initial']
+
+
+def info():
+    """Info message."""
+    return "Resize uuid columns on workflows tables."
+
+
+def do_upgrade():
+    """Implement your upgrades here."""
+    with op.batch_alter_table("bwlWORKFLOW") as batch_op:
+        batch_op.alter_column(
+            column_name='uuid',
+            type_=db.UUID(), nullable=False
```

Comment:
It can contain integers?
Because it's connected as ForeignKey with other table that use UUID.
We shound choiceone of this solutions:
- [ ] remove the foreign key
- [ ] convert all in a uuid field
- [ ] convert other connected fields as string as well